esx_service/cli/vmdkops_admin_test.py

```diff
@@ -116,125 +116,128 @@ def test_policy_ls(self):
 
     def test_policy_ls_badargs(self):
         self.assert_parse_error('policy ls --name=yo')
+    
+    # NOTE: "tenant" is renamed to "vm-group", but we only change it in command line
+    # all the function name remain unchanged
 
     def test_tenant_create(self):
-        args = self.parser.parse_args('tenant create --name=tenant1 --vm-list vm1,vm2'.split())
+        args = self.parser.parse_args('vm-group create --name=vm-group1 --vm-list vm1,vm2'.split())
         self.assertEqual(args.func, vmdkops_admin.tenant_create)
-        self.assertEqual(args.name, 'tenant1')
+        self.assertEqual(args.name, 'vm-group1')
         self.assertEqual(args.vm_list, ['vm1', 'vm2'])
 
     def test_tenant_create_missing_option_fails(self):
-        self.assert_parse_error('tenant create')
+        self.assert_parse_error('vm-group create')
```

Comment:
I think it would be much better to have a global definition and use a VMGROUP name everywhere. 
So this line would be `self.assert_parse_error(VMGROUP + ' create')`

This comment applies to ALL changes.